deployment/cluster-role.yaml

```diff
@@ -9,3 +9,9 @@ rules:
 - apiGroups: ["aks-periscope.azure.github.com"]
   resources: ["diagnostics"]
   verbs: ["get", "watch", "list", "create", "patch"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["access.smi-spec.io", "specs.smi-spec.io", "split.smi-spec.io"]
+  resources: ["*"]
+  verbs: ["get", "list", "watch"]
```

Comment:
Hi @arnaud-tincelin, I've updated the kustomize resource file so that CRDs and SMI CRDs (both required) are added to the periscope cluster role permissions.
`deployment/cluster-role.yaml`